modules/compiler/targets/host/CMakeLists.txt

```diff
@@ -188,8 +188,13 @@ if(CA_HOST_CROSS_COMPILERS)
     target_compile_definitions(compiler-host PRIVATE
     CA_HOST_TARGET_${HOST_ARCH_UPPER}_CPU="${CA_HOST_TARGET_${HOST_ARCH_UPPER}_CPU}")
   endif()
+  set(CA_HOST_TARGET_${HOST_ARCH_UPPER}_FEATURES_DESC
+    "Feature list that host ${HOST_ARCH_UPPER} should enable or disable as a comma separated + or - list")
+  if(HOST_ARCH_UPPER STREQUAL RISCV32 OR HOST_ARCH_UPPER STREQUAL RISCV64)
+    string(APPEND CA_HOST_TARGET_${HOST_ARCH_UPPER}_FEATURES_DESC " e.g. '+v,+zfh'")
```

Comment:
I get where you are coming from, but the intention was to show the form of it. However, if you found it confusing as is, then I'll accept your change.

Reply:
If the arch name hadn't been wrong, I probably would not have found it nearly as confusing, but looking at what we end up with (other variables omitted):

```diff
-//Feature list that host RISCV64 should enable or disable as a
-// comma separated + or - list e.g. '+v,+zfh'
+//Feature list that host AARCH64 should enable or disable as a
+// comma separated + or - list
 CA_HOST_TARGET_AARCH64_FEATURES:STRING=

 //Feature list that host RISCV64 should enable or disable as a
 // comma separated + or - list e.g. '+v,+zfh'
 CA_HOST_TARGET_RISCV64_FEATURES:STRING=

-//Feature list that host RISCV64 should enable or disable as a
-// comma separated + or - list e.g. '+v,+zfh'
+//Feature list that host X86_64 should enable or disable as a comma
+// separated + or - list
 CA_HOST_TARGET_X86_64_FEATURES:STRING=
```

This does manage to still show off the form IMO, as these variables are all closely together. But I'd also be happy with a way of showing off the form on each variable if we can find one that avoids giving an example that is never valid for some of the variables.

Reply:
I think it is okay as it's documented in the developer guide.